ecl/hql/hqlutil.hpp

```diff
@@ -197,6 +197,8 @@ extern HQL_API IHqlExpression * createTransformForField(IHqlExpression * field,
 extern HQL_API IHqlExpression * convertScalarToRow(IHqlExpression * value, ITypeInfo * fieldType);
 extern HQL_API bool splitResultValue(SharedHqlExpr & dataset, SharedHqlExpr & attribute, IHqlExpression * value);
 
+//Is this really dependent on a parameter - expr->isFullyBound() can give false positives.
```

Comment:
Additionally, if expr->isFullyBound() can give false positives, I don't see how this has been dealt with, or is the point of the comment, that it has not?
i.e. both isDependentOnParameter and isDependent check and return on expr->isFullyBound == true